eve_sqlalchemy/tests/patch.py

```diff
@@ -27,6 +28,7 @@ def test_patch_defaults(self):
                                  {'title': None}, None)
         super(TestPatch, self).test_patch_defaults()
 
+    @pytest.mark.xfail(True, run=False, reason='not applicable to SQLAlchemy')
     def test_patch_defaults_with_post_override(self):
```

Comment:
You can remove the implementation of the tests too, we will not need it anymore. See e.g. `test_patch_null_objectid`.